### PR TITLE
Storm 1290:port backtype.storm.local-state-test to java

### PR DESCRIPTION
**Pull request by kamleshbhatt**
These changes are in continuation of pull request
https://github.com/apache/storm/pull/2074

Please review.

### CODE REVIEW

File: storm-server/src/test/java/org/apache/storm/LocalStateTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
+package org.apache.storm;
```
Comment (srdo) [inline storm-server/src/test/java/org/apache/storm/LocalStateTest.java]: You need to add the Apache license to the top of this file. You can copy it from any of the other files in this directory.

File: storm-server/src/test/java/org/apache/storm/LocalStateTest.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    }
+
+    @Test
+    public void testEmptyState() throws IOException {
+        TmpPath tmp_dir = new TmpPath();
+        String dir = tmp_dir.getPath();
+        LocalState ls = new LocalState(dir);
+        GlobalStreamId gs_a = new GlobalStreamId("a", "a");
+        FileOutputStream data = FileUtils.openOutputStream(new File(dir,"12345"));
+        FileOutputStream version =FileUtils.openOutputStream(new File(dir,"12345.version"));
```
Comment (srdo) [inline storm-server/src/test/java/org/apache/storm/LocalStateTest.java]: Missing space after =. I think checkstyle may complain about this.

**Comment by kamleshbhatt** [comment]
Modified as per review comments.

**Comment by HeartSaVioR** [comment]
+1

**Comment by HeartSaVioR** [comment]
Merged via b3389abbfdc37fb89d838547769703fedcd82d27

**Pull request closed without merging**